DOL.WHD.Section14c.Web/src/modules/pages/landingPageController.js

```diff
@@ -59,54 +42,66 @@ module.exports = function(ngModule) {
+            stateService.applicationId = result.data.ApplicationId;
+            $location.path('/section/assurances');
+            autoSaveService.start();
+          }).catch(function(error) {
+            $scope.handleApplicationLoadError(error.data);
+          });
+      }
+    };
+
     $scope.handleApplicationLoadError = function(message) {
       $scope.applicationLoadError.status = true;
       if(message) {
         $scope.applicationLoadError.message = message;
       }
     };
 
-    $scope.loadUserInfo = function(index) {
-      stateService.employerId = $scope.organizations[index].employer.id;
-      stateService.applicationId = $scope.organizations[index].applicationId;
-      stateService.ein = $scope.organizations[index].ein;
-      stateService.employerName = $scope.organizations[index].employer.legalName;
+    $scope.loadUserInfo = function(employerId, ein, employerName) {
+      stateService.employerId = employerId;
+      stateService.ein = ein;
+      stateService.employerName = employerName;
       return;
     }
 
     $scope.init = function () {
       apiService.userInfo(stateService.access_token).then(function(result) {
-        $scope.organizations = result.data.organizations;
-        $scope.applicationList = [];
         result.data.organizations.forEach(function(element) {
+          $scope.loadUserInfo(element.employer.id, element.employer.ein, element.employer.legalName);
           var organization = {
-            applicationId: element.applicationId,
-            employerId: element.employer.ein,
+            ein: element.employer.ein,
+            employerId: element.employer.id,
             employerName: element.employer.legalName,
             createdAt: dateFilter(element.createdAt) + " at " + new Date(element.createdAt).toLocaleTimeString(),
             lastModifiedAt: dateFilter(element.lastModifiedAt) + " at " + new Date(element.lastModifiedAt).toLocaleTimeString(),
-            applicationStatus: element.applicationStatus.name,
             employerAddress: element.employer.physicalAddress.streetAddress + " " + element.employer.physicalAddress.city + ", " + element.employer.physicalAddress.state + " " + element.employer.physicalAddress.zipCode
           }
-
-          if(element.applicationStatus.name === "New") {
-            organization.action = "Start"
+          if(element.applicationId && element.applicationStatus) {
+            organization.applicationId = element.applicationId;
+            organization.applicationStatus = element.applicationStatus.name;
+             if(element.applicationStatus.name === "Submitted") {
+              organization.action = "Download";
+              $scope.submittedApplications.push(organization);
+            }
           }
-          else if (element.applicationStatus.name === "InProgress") {
-            organization.action = "Continue"
-          } else {
-            organization.action = "Download"
+          if(organization.action != "Download") {
```

Comment:
This seems to be making the assumption that only one application will ever not be "Submitted" - which also seems like it's the right assumption, but I think it'd be useful to put a comment here just calling that out.  Otherwise someone in the future might look at this and think there's a potential bug.